### DISCUSSION TITLE
How about converting JBang directive to Triple-Slash?

### DISCUSSION THREAD

**Discussion by linux-china** [jbangdev/jbang#1174] [Ideas]
`//` is normal JVM languages' comment and used a lot in code,  but `///` is used in JVM languages rarely and better for directives.   Triple-Slash directives are easy to parse and process, and it also has been used in TypeScript and Rust. 

```java
///usr/bin/env jbang "$0" "$@" ; exit $?
///JAVA 11
///DEPS org.apache.commons:commons-lang3:3.12.0
///DEPS org.apache.commons:commons-text:1.9
/// this is simple Java Hello

import org.apache.commons.lang3.StringUtils;

public class Hello {
    public static void main(String... args) {
        StringUtils.center("====", 20);
    }
}
```

Just my opinion for discussion, and `//` still ok to me.

**Comment by maxandersen** [comment]
In rust they are still comments but special ones. Not the same. 

The syntax is '//uppercasecommandorsymbols input'. 

That should be sufficient to not collide with anything and used to extract is it not ?

**Reply by quintesse** [reply]
I also feel that a drastic change like this , which would break almost **all** Jbang scripts (!!!), needs an equally drastic reason for doing so. Even if we're not at 1.0 yet I feel that a breaking change like this would create a lot of bad feelings in our user base.
